### PR TITLE
Type definitions for weaveid

### PR DESCRIPTION
**Pull request by martonlederer**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## weaveid ([<kbd>unpkg</kbd>](https://unpkg.com/browse/weaveid@latest/))
was missing the following properties:
1. closeLoginModal
1. weaveIDInit
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a5bff50e26415681b9b99516d87e445f7f7c7dae
</p>

File: types/weaveid/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    closeLoginModal: typeof closeModal;
+    openLoginModal: typeof openLoginModal;
+    getAddress: typeof getAddress;
+    getWallet: typeof getWallet;
+    WeaveID: any;
+    init: typeof init;
+  }
+}
+
+export function closeModal(): void;
```
Comment (amcasey) [inline types/weaveid/index.d.ts:19]: Should this be `closeLoginModal`?

**Review by amcasey — CHANGES_REQUESTED** [review]
LGTM but I think there's a typo.

**Comment by martonlederer** [comment]
Thanks @amcasey, fixed it

File: types/weaveid/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Type definitions for use-color-scheme 1.0
```
Comment (peterblazejewicz) [inline types/weaveid/index.d.ts:1]:
```suggestion
// Type definitions for weaveid 1.0
```

Comment (martonlederer) [reply types/weaveid/index.d.ts:1]: Uh lol thanks

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@martonlederer 
There is a slight mismatch with the published version on unpkg, but this is probably due to insconsistency in source itself (mismatch of aliases for global and exports),
Thx!

**Review by amcasey — APPROVED** [review]

**Comment by martonlederer** [comment]
Ready to merge

**Pull request merged**